### PR TITLE
feat: lowercase and remove special chars on parameter files

### PR DESCRIPTION
**Pull request by cryptonemo**
Updates name output where used in parameter file path generation.

### CODE REVIEW

File: storage-proofs/core/src/hasher/sha256.rs [hunk 1]
```diff
@@ -24,7 +24,7 @@ impl Hasher for Sha256Hasher {
     type Function = Sha256Function;
 
     fn name() -> String {
-        "Sha256Hasher".into()
+        "sha256hasher".into()
```
Comment (dignifiedquire) [inline storage-proofs/core/src/hasher/sha256.rs:27]: lets use `sha256_hasher` for better readability

File: storage-proofs/core/src/hasher/poseidon.rs [hunk 2]
```diff
@@ -28,7 +28,7 @@ impl Hasher for PoseidonHasher {
     type Function = PoseidonFunction;
 
     fn name() -> String {
-        "PoseidonHasher".into()
+        "poseidonhasher".into()
```
Comment (dignifiedquire) [inline storage-proofs/core/src/hasher/poseidon.rs:31]: lets use `poseidon_hasher` for better readability

**Comment by dignifiedquire** [comment]
sad CI is sad

**Comment by cryptonemo** [comment]
> sad CI is sad

CI doesn't like that we renamed parameter files on it ;-)

**Comment by cryptonemo** [comment]
> > sad CI is sad
> 
> CI doesn't like that we renamed parameter files on it ;-)

@dignifiedquire What's the proper way to get new params generated on CI for this kind of thing?  The test *should* pass when the params are found.  Also, are there any other changes required before params are regenerated, and if so, maybe those could be merged here so we can do them all at once?

**Comment by dignifiedquire** [comment]
@cryptonemo you need to bump the cache versions. Both in circleci and the main cache identifier, as this is a breaking change as far as the paramchache is concerend

**Comment by cryptonemo** [comment]
@dignifiedquire There's still something here I'm not understanding :-(

**Pull request merged**